### PR TITLE
[WIP] AI Unjanking

### PR DESCRIPTION
**Pull request by jknpj**
I never asked for this.
I already regret this.
And we've got a long way to go.

### CODE REVIEW

**Comment by jknpj** [comment]
Goals:
* Organize the code a little
* Make it somehow more OOP with cyborg code(Looking at you, alarms)
* Give AIs proper access, like happened with cyborgs, to futureproof it against oversights(FEATURES)
* Find and remove the ancient copypasted cancer from its bosson
* Fix its verb insanity
* Kill myself

**Comment by SonixApache** [comment]
god speed

File: __DEFINES/silicon.dm [hunk 1]
```diff
@@ -1,3 +1,65 @@
+//All AIs in the world
+var/list/ai_list = list()
+
+//AI Icons
+var/global/list/possible_ai_icon_states = list(
```
Comment (FatCat1978) [inline __DEFINES/silicon.dm:5]: There's surely a better way of doing this than just hardcoding absolutely every icon state. you could probably just use icon_states()

Comment (jknpj) [reply __DEFINES/silicon.dm:5]: I'd have to change AI.dmi to only have AI icons though, still a nice idea

**Pull request closed without merging**